Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -58,6 +58,18 @@ struct TestCommand: AsyncParsableCommand {
         help: "Path where test result bundle will be saved."
     )
     var resultBundlePath: String?
+    
+    @Option(
+        name: .long,
+        help: "Tests will run <number> of times."
+    )
+    var testIterations: Int?
+    
+    @Flag(
+        name: .long,
+        help: "Tests will retry on failure. May be used in conjunction with -test-iterations <number>, in which case <number> will be the maximum number of iterations. Otherwise, a maximum of 3 is assumed."
+    )
+    var retryTestsOnFailure: Bool = false
```

Comment:
What about having a single `retryCount` parameter (with default `0`)?

We would have something like:
```
tuist test --retry-count 3
```

instead of
```
tuist test --test-iterations 3 --retry-tests-on-failure true
```
?

Reply:
Yeah that works for me!